jupyter_server/serverapp.py

```diff
@@ -1968,7 +1968,7 @@ def init_event_logger(self):
         ]
         for schema_id in schema_ids:
             # Get the schema path from the schema ID.
-            rel_schema_path = schema_id.lstrip(JUPYTER_SERVER_EVENTS_URI) + ".yaml"
+            rel_schema_path = schema_id.replace(JUPYTER_SERVER_EVENTS_URI + "/", "") + ".yaml"
```

Comment:
Nice catch! Thanks, @lresende 

I don't think the trailing slash is needed here, since the pathlib object in the next line can handle redundant slashes. In my experience, munging file/url paths is always a pain. 😅 

 
```suggestion
            rel_schema_path = schema_id.replace(JUPYTER_SERVER_EVENTS_URI, "") + ".yaml"
```